go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/ceph/ceph-csi
 
-go 1.22.7
+go 1.23.4
```

Comment:
using 1.23.4 will have a downstream problem, please stick to 1.23.0 for some time if 1.23.x is required